backend/routers/transcribe_v2.py

```
@@ -96,26 +98,26 @@ def _has_speech(data, sample_rate):
 
     async def _trigger_create_memory_with_delay(delay_seconds: int, finished_at: datetime):
-        # print('memory_creation_timer', delay_seconds)
+        # print('memory_creation_timer', delay_seconds, uid)
         try:
             await asyncio.sleep(delay_seconds)
 
             # recheck session
             memory = retrieve_in_progress_memory(uid)
             if not memory or memory['finished_at'] > finished_at:
-                print(f"_trigger_create_memory_with_delay not memory or not last session")
+                print(f"_trigger_create_memory_with_delay not memory or not last session", uid)
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)